docs-new/docs/graphql/core/api-reference/syntax-defs.mdx

```diff
@@ -766,232 +700,181 @@ String: String } } \]
 | timeout                | false    | Integer                                                                      | Number of seconds to wait for response before timing out. Default: 30                                                                   |
-| request_transform      | false    | `RequestTransformation`                                                      | Request Transformation to be applied to this Action's request                                                                           |
+| request_transform      | false    | [RequestTransformation](#requesttransformation)                                                      | Request Transformation to be applied to this Action's request                                                                           |
 
-## InputArgument
+## InputArgument {#inputargument}
 
 | Key  | Required | Schema                      | Description          |
 |------|----------|-----------------------------|----------------------|
 | name | true     | text                        | Name of the argument |
-| type | true     | `GraphQLType <GraphQLType>` | Type of the argument |
-
-<div class="note">
-
-<div class="title">
+| type | true     | [GraphQLType](#graphqltype) | Type of the argument |
 
-Note
-
-</div>
+:::info Note
 
 The `GraphQL Types` used in creating an action must be defined before
-via `Custom Types <metadata_api_custom_types>`
+via [Custom Types](/graphql/core/api-reference/metadata-api/custom-types.mdx)
 
-</div>
+:::
 
-## ComputedFieldDefinition
+## ComputedFieldDefinition {#computedfielddefinition}
 
 | Key              | Required | Schema                        | Description                                                                                                                                             |
 |------------------|----------|-------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
-| function         | true     | `FunctionName <FunctionName>` | The SQL function                                                                                                                                        |
+| function         | true     | [FunctionName](#functionname) | The SQL function                                                                                                                                        |
 | table_argument   | false    | String                        | Name of the argument which accepts a table row type. If omitted, the first argument is considered a table argument                                      |
 | session_argument | false    | String                        | Name of the argument which accepts the Hasura session object as a JSON/JSONB value. If omitted, the Hasura session object is not passed to the function |
 
-## Function Configuration
+## Function Configuration {#functionconfiguration}
```

Comment:
missed casing in id

`{#function-configuration}`

please update respective usage

Reply:
fixed